webapp/src/containers/Rhs/projectDetails/index.tsx

```diff
@@ -163,22 +176,26 @@ const ProjectDetails = (projectDetails: ProjectDetails) => {
                 description='Are you sure you want to delete this subscription?'
                 title='Confirm Delete Subscription'
             />
-            {isLoading && <LinearLoader/>}
-            <div className='d-flex'>
+            {isLoading && <LinearLoader extraClass='top-0'/>}
```

Comment:
`top-0` doesn't specify the meaning of what this class is used for. Can we name it so that it is more descriptive?

Reply:
I think it specifies because top-0 means o from top similar to margin-0

what you think for this?

Reply:
okay, seems fine now. I didn't understand why `top-0` was used.